features/apimgt/org.wso2.carbon.apimgt.rest.api.store.feature/pom.xml

```diff
@@ -167,6 +167,7 @@
             <plugin>
                 <groupId>io.swagger</groupId>
                 <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <version>2.3.1</version>
```

Comment:
Please move the version to the root pom. It might be difficult to update in multiple places when we update the version in future.

Reply:
Can you also add this version into admin rest api pom as well? I think not having the version here is the reason for the issue @msm1992 also got when generating docs. Thanks @shaniR !